crates/biome_fs/src/path.rs

```diff
@@ -41,9 +41,30 @@ pub enum FileKind {
 }
 
 #[derive(Debug, Clone, Hash, Ord, PartialOrd, Eq, PartialEq, Default)]
-#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(from = "Vec<FileKind>", into = "Vec<FileKind>")
```

Comment:
Just wondering: why not `SmallVec`?

Reply:
Because I don't know if it works with WASM. We can reconsider once though

Reply:
Done